flight_phase_files/HALO_RF03_20200124_info.yaml

```
@@ -19,59 +19,115 @@ remarks:
-  end: 2020-01-24 12:15:10
-- kind: circle
-  name: c3
-  start: 2020-01-24 12:15:10
-  end: 2020-01-24 13:10:03
-- kind: circle_break
-  name: cb3to4
-  start: 2020-01-24 13:10:03
-  end: 2020-01-24 13:30:05
-- kind: circle
-  name: c4
-  start: 2020-01-24 13:30:05
-  end: 2020-01-24 14:25:07
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-24 14:25:07
-  end: 2020-01-24 14:40:08
-- kind: circle
-  name: c5
-  start: 2020-01-24 14:40:08
-  end: 2020-01-24 15:35:05
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-24 15:35:05
-  end: 2020-01-24 15:50:06
-- kind: circle
-  name: c6
-  start: 2020-01-24 15:50:06
-  end: 2020-01-24 16:45:10
-- kind: straight_leg
-  name: leg1_outbound_NTAS
+- kinds:
+  - circle
+  name: Circle-1
+  irregularities: null
+  segment_id: HALO-0124c1
+  start: 2020-01-24 09:49:17
+  end: 2020-01-24 10:46:35
+  good_dropsondes: null
+- kinds:
+  - circle_break
+  name: Circle break between circle-1 and circle-2
+  irregularities: sonde intended for circle 1 included here
+  segment_id: HALO-0124cb1
+  start: 2020-01-24 10:46:35
+  end: 2020-01-24 10:59:08
+- kinds:
+  - circle
+  name: Circle-2
+  irregularities: null
+  segment_id: HALO-0124c2
+  start: 2020-01-24 10:59:08
+  end: 2020-01-24 11:57:07
+  good_dropsondes: null
+- kinds:
+  - circle_break
+  name: Circle break between circle-2 and circle-3
+  irregularities: sonde intended for circle 2 included here
+  segment_id: HALO-0124cb2
+  start: 2020-01-24 11:57:07
+  end: 2020-01-24 12:14:11
+- kinds:
+  - circle
+  name: Circle-3
+  irregularities: null
+  segment_id: HALO-0124c3
+  start: 2020-01-24 12:14:11
+  end: 2020-01-24 13:13:29
+  good_dropsondes: null
+- kinds:
+  - circle_break
+  name: Circle break between circle-3 and circle-4
+  irregularities: null
+  segment_id: HALO-0124cb3
+  start: 2020-01-24 13:13:29
+  end: 2020-01-24 13:29:04
+- kinds:
+  - circle
+  name: Circle-4
+  irregularities: null
+  segment_id: HALO-0124c4
+  start: 2020-01-24 13:29:04
+  end: 2020-01-24 14:28:38
+  good_dropsondes: null
+- kinds:
+  - circle_break
+  name: Circle break between circle-4 and circle-5
+  irregularities: null
+  segment_id: HALO-0124cb4
+  start: 2020-01-24 14:28:38
+  end: 2020-01-24 14:39:08
+- kinds:
+  - circle
+  name: Circle-5
+  irregularities: null
+  segment_id: HALO-0124c5
+  start: 2020-01-24 14:39:08
+  end: 2020-01-24 15:38:38
+  good_dropsondes: null
+- kinds:
+  - circle_break
+  name: Circle break between circle-5 and circle-6
+  irregularities: null
+  segment_id: HALO-0124cb5
+  start: 2020-01-24 15:38:38
+  end: 2020-01-24 15:49:06
+- kinds:
+  - circle
+  name: Circle-6
+  irregularities: null
+  segment_id: HALO-0124c6
+  start: 2020-01-24 15:49:06
+  end: 2020-01-24 16:48:19
+  good_dropsondes: null
+- kinds:
+  - circling
+  name: Circling period - Circle 1 to 6
+  irregularities: null
+  segment_id: HALO-0124o1
+  start: 2020-01-24 09:49:17
+  end: 2020-01-24 16:48:19
+  good_dropsondes: null
```

In case the no-launch-detect sondes shall be subtracted, the number would be 70 (see comments above).
```suggestion
  good_dropsondes: 73
```